### ISSUE TITLE
Training setting for different backbones

### ISSUE THREAD

**Issue by xiaodongww** [michuanhaohao/reid-strong-baseline#45]
Hi, Thanks for your great work!

There is a quick question for the performance comparison of different network backbones provided in your repo. Do you use the same settings(e.g. training epochs, initial learning rate, when to deacy lr) for all backbones?   Thank you!

**Comment by shaoniangu** [comment]
Hi. Yes, the training settings are all the same.
